client/rpmtrans.c

```diff
@@ -211,13 +211,104 @@ TDNFPopulateTransaction(
     goto cleanup;
 }
 
+uint32_t
+TDNFDetectPreTransFailure(
+    rpmts pTS,
+    char *pszError
+    )
+{
+    uint32_t dwError = 0;
+    rpmtsi pi = NULL;
+    rpmte pte = NULL;
+    char *pszToken;
+    int i = 0;
+    char *pszPkgName = NULL;
+    char *pszSymbol = NULL;
+    char *pszVersion = NULL;
+    char *pszEVR = NULL;
+    char *pszCachePkgName = NULL;
+    char *pszCachePkgEVR = NULL;
+
```

Comment:
better to check for pszError before we use it? If it is NULL or empty string

Reply:
Done. Thanks.